omx/gstomxaudiodec.c

```diff
@@ -762,9 +749,6 @@ gst_omx_audio_dec_stop (GstAudioDecoder * decoder)
 
   GST_DEBUG_OBJECT (self, "Stopping decoder");
 
-  gst_omx_port_set_flushing (self->dec_in_port, 5 * GST_SECOND, TRUE);
```

Comment:
Why is this no longer required?

Reply:
Omx component and element tear down fix (as explained in comment from above)